File: Python/lista2 - estruturas de repeticao/exercicio_8.py
i = 1
age = 0
sum_age = 0
average = 0

while i != 0:
    age = int(input('Informe sua idade: '))

    if age != 0:
        sum_age += age
    else:
        break

    i += 1

average = sum_age / (i - 1)

print(f'Média das idades: {average} ')

File: C/lista2 - estrutura de repeticao/exercicio_8.c
#include <conio.h>
#include <stdio.h>

int main() {
	int age, i = 1, sumAge = 0;
	float average = 0;
	
	for (i= 0; i!= 0; i++) {
		printf("Informe uma idade: ");
		scanf("%d", &age);
		
		if (age != 0) {
			sumAge += age;
		} else {
			break;
		}
	}
	
	average = (float) sumAge / i;
	
	printf("Media das idades: %f", average);
	
	getch();
	return 0;
}